tools/bioimaging/bioimage_inference.xml

```diff
@@ -46,16 +48,50 @@
             <param name="input_image_file" value="input_nucleisegboundarymodel.png"/>
             <param name="input_image_input_size" value="256,256,1,1"/>
+            <param name="input_image_input_axes" value="bcyx"/>
             <output name="output_predicted_image" file="output_nucleisegboundarymodel.tif" compare="sim_size" delta="100" />
             <output name="output_predicted_image_matrix" file="output_nucleisegboundarymodel_matrix.npy" compare="sim_size" delta="100" />
         </test>
+        <test>
+            <param name="input_imaging_model" value="input_imaging_model.zip" location="https://uk1s3.embassy.ebi.ac.uk/public-datasets/bioimage.io/emotional-cricket/1.1/files/torchscript_tracing.pt"/>
+            <param name="input_image_file" value="input_image_file.tif" location="https://uk1s3.embassy.ebi.ac.uk/public-datasets/bioimage.io/emotional-cricket/1.1/files/sample_input_0.tif"/>
+            <param name="input_image_input_size" value="128,128,100,1"/>
+            <param name="input_image_input_axes" value="bczyx"/>
+            <output name="output_predicted_image" file="output_3d-unet-arabidopsis-apical-stem-cells.tif" compare="sim_size" delta="100" />
+            <output name="output_predicted_image_matrix" file="output_3d-unet-arabidopsis-apical-stem-cells.npy" compare="sim_size" delta="100" />
+        </test>
+        <test>
+            <param name="input_imaging_model" value="input_imaging_model.zip" location="https://uk1s3.embassy.ebi.ac.uk/public-datasets/bioimage.io/emotional-cricket/1.1/files/torchscript_tracing.pt"/>
+            <param name="input_image_file" value="input_3d-unet-arabidopsis-apical-stem-cells.png" location="https://uk1s3.embassy.ebi.ac.uk/public-datasets/bioimage.io/emotional-cricket/1.1/files/raw.png"/>
+            <param name="input_image_input_size" value="128,128,100,1"/>
+            <param name="input_image_input_axes" value="bczyx"/>
+            <output name="output_predicted_image" file="output_3d-unet-arabidopsis-apical-stem-cells.tif" compare="sim_size" delta="100" />
+            <output name="output_predicted_image_matrix" file="output_3d-unet-arabidopsis-apical-stem-cells.npy" compare="sim_size" delta="100" />
+        </test>
+        <test>
+            <param name="input_imaging_model" value="input_imaging_model.zip" location="https://uk1s3.embassy.ebi.ac.uk/public-datasets/bioimage.io/organized-badger/1/files/weights-torchscript.pt"/>
+            <param name="input_image_file" value="input_platynereisemnucleisegmentationboundarymodel.tif" location="https://uk1s3.embassy.ebi.ac.uk/public-datasets/bioimage.io/organized-badger/1/files/sample_input_0.tif"/>
+            <param name="input_image_input_size" value="256,256,32,1"/>
+            <param name="input_image_input_axes" value="bczyx"/>
+            <output name="output_predicted_image" file="output_platynereisemnucleisegmentationboundarymodel.tif" compare="sim_size" delta="100" />
```

Comment:
@anuprulez you can use the new image_diff here https://docs.galaxyproject.org/en/latest/dev/schema.html#id117